libiop/algebra/field_subset/field_subset.hpp

```diff
@@ -64,7 +64,6 @@ class field_subset {
 
     FieldT generator() const;
 
-    const FieldT& offset() const;
     const FieldT shift() const;
```

Comment:
We should probably open an issue for understanding when to use FieldT vs FieldT&. There isn't a unified approach to this taken throughout the codebase atm.